### DISCUSSION TITLE
placeholder 设置无效

### DISCUSSION THREAD

**Discussion by yanhuacuo** [walinejs/waline#176] [Q&A] [answered]
Waline v0.14.8 中，虽然本地已经设了「placeholder」，但是并没有替换……

**Comment by lizheming** [comment, marked as answer]
可以提供下线上地址我去 check 一下。

**Reply by yanhuacuo** [reply]
因为「placeholder」无效，只好退而使用本地化的 js,，改了just go go在用（「评论区已开启广告机过滤」）。

当前显示为「评论区已开启广告机过滤」，但其实hexo配置里写的是「已开启广告机过滤」。

另外，「requireFields」虽然用了['nick','mail']，也没生效，依然可以不填邮箱评论。

https://wubi98.gitee.io/2019/05/16/2019-05-16-005.98chr/

**Reply by lizheming** [reply]
<img width="851" alt="图片" src="https://user-images.githubusercontent.com/424491/111891033-2e00dc80-8a2a-11eb-92c7-a40443b55fde.png">

看了下源码，应该是你的主题配置有问题，我看配置都没有传入进来（otherData 那的 false）。

**Reply by yanhuacuo** [reply]
> <img alt="图片" width="851" src="https://user-images.githubusercontent.com/424491/111891033-2e00dc80-8a2a-11eb-92c7-a40443b55fde.png">
> 
> 看了下源码，应该是你的主题配置有问题，我看配置都没有传入进来（otherData 那的 false）。

感谢指点，我再查一查原因。
